README.md

```diff
@@ -21,13 +21,13 @@ To get your own Glitch-hosted Probot up-and-running, follow these steps. If you
 
 2. Click the **Install** tab, and install your app into one of your repositories.
 
-3. Click the **New File** button (at left) and type `.data/private-key.pem`. Then click **Add File**. Open a terminal, and from your download folder run `cat my-app-name.2018-06-20.private-key.pem | pbcopy` (except using your app's name and today's date). In the new file in Glitch, paste the contents of the clipboard.
+3. Click the **New File** button (at left) and type `.data/private-key.pem`. Then click **Add File**. Now we need to copy the key to the clipboard. You can [follow the directions here](https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/) to find out how to do that on your system, bearing in mind that your private key is in your Downloads folder, and will be named like `my-app-name.2018-06-20.private-key.pem `. In the new file in Glitch, paste the contents of the clipboard.
```

Comment:
It's great that this is becoming more OS-agnostic, but could the ssh-key instructions be a bit misleading/confusing here?